app/src/timebar/components/DatePicker.jsx

```diff
@@ -24,6 +25,7 @@ class DatePicker extends Component {
         </div>
         <ReactDatePicker
           fixedHeight
+          calendarClassName={classnames(['calendar', { endDatePicker: this.props.label === 'end' }])}
```

Comment:
But this should be only applied to the calendar on the right of the screen. Maybe rightCalendar could be more explicit about the styling